### ISSUE TITLE
suggestion to restructure json file locations

### ISSUE THREAD

**Issue by robertoostenveld** [donders-research-data-management/rdm-ontology#5]
Right now the external_urls is maintained at the toplevel, but the ethics_boards and publication_schema are one level down in "vocabularies".

Can we make them consistent, i.e. have the respective files at the same level in the repository?

**Comment by robertoostenveld** [comment]
pinging @hurngchunlee

**Comment by hurngchunlee** [comment]
I see the external_urls is an "index-like" file of the repository; while various contents are located in different subfolders.   The index file instructs the CMS code to load contents from the specified locations.

Thus, I don't see the reason that ethics_boards and publication_schema should be moved out from the "vocabularies" subfolder.

**Comment by robertoostenveld** [comment]
It is not the specific content of this particular that triggered me to think about this. 

I am thinking of the BSI (and other) data stewards that are going to be maintaining the content. There are different types of content in this repository: json+schema (parsed by the web frontend code), html (inserted by the web fronted code), and utilities.

In the upcoming introductions to the data stewards we (i.e. me and Eric) should be able to explain how *they* would change the static content. For that it would be good if it were refactored a bit. 

We may even want to rename it to content-donders, thereby making space for the content-bis and possibly content-languageininteraction forks/copies.

From @paulspeijers "development view" document

```
Configurable content

Configurable content can be managed by system administrators in order to change the information shown within the web portal, or returned from within the service layer. Examples are controlled vocabularies such as MeSH and SFN, publication systems, data usage agreements, ethical review boards. Furthermore texts within the web portal are configurable, such as the footer and homepage texts, as well as the locations of all contents.
Configurable content should not contain sensitive information because DR exposes it publicly.
```

But note that it is not the system administrator that decides on the content, it is the research director (who delegates this to the data steward). The system administrator reviews the suggested changes (on technical compatibility) and merges it into the release.

**Comment by hurngchunlee** [comment]
I also think that we need a documentation (e.g. a README) for the data-stewards.  The current README is very technical.

**Comment by hurngchunlee** [comment]
I propose the following organisation:

- rename external_url.json/schema to index.json/schema
- organise HTML snippets into a "web_snippets" folder; those are "login", "logout", "email", "footer", "homepage", "messages", "privacy"
- organise controlled vocabularies into a "controlled_vocabularies" folder; those are "dua", "keywords", "ethical_review_boards", "publication_systems" ... on issue here is that "dua" consists of both controlled vocabulary and HTML snippet.

**Comment by robertoostenveld** [comment]
I don't see the advantage of the rename of external_urls into index. Then better keep it at the root level with its existing name. I do like the other two suggestions.

I would call the 2nd "static_content". Would you want that one to have subdirectories, or all snippets on the same level? The only subdirectory that I see making sense (containing multiple snippets) is the one with the DUAs.

I would call the 3rd "vocabularies". We don't have non-controlled vocabularies, so it does not have to be mentioned that ours are "controlled".

**Comment by robertoostenveld** [comment]
Regarding static content, we might want to change it such that 

https://data.donders.ru.nl/doc/xxx

will always embed the snippet xxx.html from the static_content directory. The snippet would be framed within the general page structure, i.e. starting with the header and menu. Idem for https://data.donders.ru.nl/doc/xxx/yyy, which embeds xxx/yyy.html

This allows for great flexibility, e.g. allowing future users of the system to embed the static help documentation in the CMS.

This would also resolve the issue that pages such as https://data.donders.ru.nl/privacy-policy?10 now have two "h1" titles. The first "Privacy Policy" and the horizontal line don't have to be there.

**Comment by robertoostenveld** [comment]
this has mostly been addressed in PR #14
